vhdbuilder/packer/install-dependencies.sh

```diff
@@ -127,6 +127,11 @@ fi
 if ! isMarinerOrAzureLinux "$OS"; then
   overrideNetworkConfig || exit 1
   disableNtpAndTimesyncdInstallChrony || exit 1
+
+  # Configure SSH service during VHD build for Ubuntu 22.10+
+  if [ "$OS" = "$UBUNTU_OS_NAME" ]; then
+    configureSSHService || echo "SSH Service configuration failed, but continuing VHD build"
```

Comment:
I'm wondering if we could turn this `echo` into a ADO Warning such that it makes the build step "yellow"